tools/downloader/pytorch_to_onnx.py

```diff
@@ -9,19 +10,28 @@
 import torch.onnx
 
 
+def is_sequence(element):
+    return isinstance(element, (list, tuple))
```

Comment:
You should use `collections.abc.Sequence` for this.

Reply:
I specially don't use `collections.abc.Sequence`, because string is sequence too. So if you use `--input-shape="[1,1,256,256],[1,2,256,256],[1,1,256,256]"` , shapes will be interpreted as sequence and integer error will be raised, but situation is not related to integers. For this reason I established sequence as (tuple, list).

Reply:
OK, good point. (To be fair, a string would get rejected by the other checks, but it's good to fail early.)